native/screens/Applicants/Applicants.jsx

```diff
@@ -74,7 +68,7 @@ const RoleList = ({ applicants, navigation, role, currProject }) => {
     return (
       <View>
         {loadMore && <Button title="Show less" onPress={() => handleToggle(toggleLoadMore)} />}
-        <Text>{role}</Text>
+        <Text style={styles.largerText}>{role}</Text>
```

Comment:
The styling for this text box is not the same color (#505050) , size (18), or margin (centered with 15px on the left + right) as depicted on the figma